.github/ISSUE_TEMPLATE/DOCUMENTATION.md

```diff
@@ -0,0 +1,51 @@
+---
+name: "📜📖📄📘 📚  Documentation enhancement"
```

Comment:
```suggestion
name: "📚 Documentation"
```

Changing this to match the format of other issue templates, with a single emoji, and the title matches the file name.